mne/epochs.py

```diff
@@ -583,7 +583,7 @@ def _detrend_offset_decim(self, epoch, verbose=None):
         return epoch
 
     def iter_evoked(self):
-        """Iterate over Evoked objects with nave=1
+        """Iterate epochs as a sequence of Evoked objects with n_epochs=1
```

Comment:
How about something like:
```
"""Iterate over epochs as a sequence of Evoked objects

The Evoked objects will each contain a single epoch (i.e. no averaging is performed).
...
```
Thus we can avoid `nave` and `n_epochs` altogether?